lib/nostrum/api/helpers.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Nostrum.Api.Helpers do
+  @moduledoc """
+  Helper functions for the Nostrum API.
+  """
```

Comment:
This module shouldn't be documented since its an implementation detail imo.
```suggestion
  @moduledoc false
```